content/docs/apps.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Third-Party Applications
+url: /docs/apps.html
+---
+<h2 id="android">Android <a class="anchor" href="#android" title="Permalink">¶</a></h2>
+
+<h3 id="microflux">Microflux <a class="anchor" href="#microflux" title="Permalink">¶</a></h3>
+
+This reader has features that cannot be provided to form a conventional web page.
+
+Features:
+- Offline persistence and sync
```

Comment:
I blank line here might be necessary. Take a look at the preview: https://deploy-preview-32--miniflux-website.netlify.app/docs/apps.html

Reply:
Thanks for the review. It has been fixed.